backend/src/test/java/com/barmjz/productivityapp/validators/EmailValidatorTest.java

```diff
@@ -0,0 +1,64 @@
+package com.barmjz.productivityapp.validators;
+
+import com.barmjz.productivityapp.user.registration.validators.EmailValidator;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class EmailValidatorTest {
+
+    private EmailValidator emailValidator;
+
+    @BeforeEach
+    void setUp() {
+        emailValidator = new EmailValidator();
+    }
+
+    @Test
+    void acceptsValidEmail(){
+        // Given
+        String email = "84-Du.M.my_124@Domain.com";
+        // When
+        boolean result = emailValidator.test(email);
+        // Then
+        assertThat(result).isTrue();
+    }
+
+    @Test
+    void rejectsEmailWithoutAtSign(){
+        // Given
+        String email = "84-DuMmy_124Domain.com";
+        // When
+        boolean result = emailValidator.test(email);
+        // Then
+        assertThat(result).isFalse();
+    }
+    @Test
+    void rejectsEmailWithoutDotAtStart(){
```

Comment:
Name of method should be change to "rejectsEmailWithDotAtStart()"

Reply:
Done.